src/applications/disability-benefits/all-claims/tests/automated-tests.puppeteer.spec.js

```diff
@@ -16,7 +16,7 @@ const testData = getTestDataSets(join(__dirname, 'data'), {
 const testConfig = {
   debug: true,
   setup: userToken => {
-    PageHelpers.initDocumentUploadMock();
+    PageHelpers.initDocumentUploadMock(userToken);
```

Comment:
When the e2e test (`automated-tests.puppeteer.spec.js`) calls `/v0/upload_supporting_evidence`, it's returning a `500 Internal Server Error` and based on the `mockapi.js` it needs a userToken to obtain a response. This will address that issue.